src/dymaptic.GeoBlazor.Core/Components/Widgets/HomeWidget.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets
```

Comment:
Please use file-scoped namespaces. (namespace with `;`, no brackets). See other files for example.

Reply:
re-scoped